README.md

```diff
@@ -1 +1,53 @@
 # Awesome-Books
+
+> Awesome books is single page application used for storing books
+
```

Comment:
- [ ] GOOD JOB 👌👏. The app is working already. However, there are some missing parts in the documentation. Please kindly add the screenshot of the app to the readme file 📃

- [ ] Let's also fix the description of your pull request. You can add a heading, a list of things you used or implemented like luxon 😃